internal/experimental/registry/client.go

```diff
@@ -144,7 +146,60 @@ func (c *Client) PushChart(ref *Reference) error {
 }
 
 // PullChart downloads a chart from a registry
-func (c *Client) PullChart(ref *Reference) error {
+func (c *Client) PullChart(ref *Reference) (*bytes.Buffer, error) {
```

Comment:
Note to reviewers... this function is internal and part of the experiment. It's ok to change it.